data-ingestion-apis/opentelemetry-metrics.mdx

```diff
@@ -1,42 +1,77 @@
 ---
-title: "Data Ingestions API"
+title: "Customizing Telemetry Ingestion"
+sidebarTitle: "OpenTelemetry"
 ---
 
 # Overview
 This guide provides an overview and setup instructions for Middleware’s OpenTelemetry Data Ingestion API. The OpenTelemetry Ingestion API has two endpoints: [metrics](#metrics-endpoint) and [logs](#logs-endpoint).
```

Comment:
We recommend using Middleware's Agent and associated SDKs to ingest data, however for more advanced users we do allow using the OpenTelemetry Protocol Exporter. Each language will have a different approach which can be an adaptation from our relevant Host Agent.

The example below is using javascript and an abstraction from our [node agent](https://github.com/middleware-labs/agent-apm-nodejs). All other agents can be found in our [middleware-labs repo](https://github.com/middleware-labs). If you need assistance with a specific language we are happy to work with you, however we do not have every approach documented at this time.